api/nakadi-event-bus-api.yaml

```diff
@@ -800,18 +800,22 @@ paths:
       security:
         - oauth2: ['nakadi.event_stream.read']
       description: |
-        Endpoint for committing offsets of the subscription. The offsets are per partition of EventType. One
-        can use this endpoint to commit the offsets for any of the partitions that are part of this
-        subscription.
-        With every commit the offsets can only increase so it is not possible to commit offset that is lower
-        than already committed offset for the appropriate partition; if such request is made - it will not
-        have any effect on the committed offsets.
+        Endpoint for committing offsets of the subscription.
+        The behavior of commit is specific to `commit_mode` and `commit_semantics` parameters of the session
+        that makes a commit. Please see details in subscription streaming endpoint specification.
       parameters:
         - name: subscription_id
           in: path
           type: string
           description: Id of subscription
           required: true
+        - name: X-Nakadi-SessionId
+          in: header
+          type: string
+          description: |
+            Id of a session (client) which makes a commit. It is not possible to make a commit from a dead or
+            none-existing session. Also the session can't commit something which was not sent to it.
```

Comment:
I think it changes the meaning. Your description says that it is not possible to commit what was not yet streamed to client. My description says that it is not possible to commit what was not streamed to client (and maybe will never be streamed to this client, like data of partition that is not delivered to this client)